node/src/database/db_initializer.rs

```diff
@@ -6,12 +6,12 @@ use crate::db_config::secure_config_layer::EXAMPLE_ENCRYPTED;
 use crate::sub_lib::accountant::{DEFAULT_PAYMENT_THRESHOLDS, DEFAULT_SCAN_INTERVALS};
 use crate::sub_lib::neighborhood::DEFAULT_RATE_PACK;
 use masq_lib::blockchains::chains::Chain;
+#[cfg(test)]
```

Comment:
I doubt that it is truly `#[cfg(test)]`, it cannot be, because you put it in `constants` in `masq_lib`, and I like it at the end of the day, but how obvious this must be that it cannot be technically done as that would be inter-crate compilation configuration with this mark, which doesn't work.   

Please remove.

Reply:
I didn't understand you objection, but removing the directive didn't seem to hurt.